util.go

```diff
@@ -0,0 +1,42 @@
+package ipldgit
+
+import (
+	"github.com/ipfs/go-cid"
+	"github.com/ipld/go-ipld-prime"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
+	mh "github.com/multiformats/go-multihash"
+)
+
+func shaToCid(sha []byte) cid.Cid {
+	h, _ := mh.Encode(sha, mh.SHA1)
+	return cid.NewCidV1(cid.GitRaw, h)
+}
+
+func cidToSha(c cid.Cid) []byte {
+	h := c.Hash()
+	return h[len(h)-20:]
```

Comment:
Hm, isn't this a bit wasteful if we only end up using part of the result? Is this to strip the multihash prefix? Might be good to document it a bit.

Reply:
right. this just uses the sha1 part of the multihash without the prefix to provide the git-compatible hash.